ksqldb-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -829,6 +829,23 @@ public void shouldNotRetryIsTopicExistsOnUnknownTopicException() {
     verify(adminClient, times(1)).describeTopics(anyCollection(), any());
   }
 
+  @Test
+  public void shouldNotRetryDescribeTopicsExistsAnyException() {
+    // When
+    when(adminClient.describeTopics(anyCollection(), any()))
+            .thenAnswer(describeTopicsResult(new UnknownTopicOrPartitionException("meh")));
+    final String topicName = "foobar";
+    final Exception e = assertThrows(
+            KafkaResponseGetFailedException.class,
+            () -> kafkaTopicClient.describeTopic(topicName, true)
+    );
+
+    // Then
+    verify(adminClient, times(1)).describeTopics(anyCollection(), any());
+    assertThat(e.getMessage(),
+            containsString("Failed to Describe Kafka Topic(s): [foobar]"));
```

Comment:
nit: leverage topicName in assertion also.
```suggestion
            containsString("Failed to Describe Kafka Topic(s): [<topicName>]"));
```